### PR TITLE
Adding New York logo to powered-by and Streams page

### PR DESCRIPTION
**Pull request by manjuapu**

### CODE REVIEW

**Comment by manjuapu** [comment]
@guozhangwang Please review.

File: 0110/streams/index.html [hunk 1]
```diff
@@ -201,39 +201,41 @@ <h3 style="max-width: 75rem;">The easiest way to write mission-critical real-tim
             </pre>
         </div>
     </div>
-    <h3 class="customer-title">See how Kafka Streams is being used</h3>
+   <h3 class="customer-title">See how Kafka Streams is being used</h3>
```
Comment (guozhangwang) [inline 0110/streams/index.html:204]: This indentation seems unintentional.

File: 0110/streams/index.html [hunk 2]
```diff
@@ -201,39 +201,41 @@ <h3 style="max-width: 75rem;">The easiest way to write mission-critical real-tim
...
     <div class="customer__cards">
          <div class="customer__card">
              <div class="customer__card__icon">
+            <img src="/{{version}}/images/icons/NYT.jpg" style="max-width:35%">
+          </div>  
+          <span class="customer__card__label">The New York Times uses Kafka and the Kafka Streams API to store and distribute, in real-time, published content to the various applications and systems that make it available to the readers.</span>
```
Comment (guozhangwang) [inline 0110/streams/index.html]: nit: ..Apache Kafka and Kafka Streams...

**Comment by manjuapu** [comment]
@guozhangwang Added Apache word

**Comment by guozhangwang** [comment]
Merged to trunk with some minor tweaks.

**Pull request closed without merging**